packages/beacon-node/src/network/processor/gossipValidatorFn.ts

```diff
@@ -134,7 +143,11 @@ export function getGossipValidatorFn(gossipHandlers: GossipHandlers, modules: Va
 
         case GossipAction.REJECT:
           metrics?.networkProcessor.gossipValidationReject.inc({topic: type});
-          logger.debug(`Gossip validation ${type} rejected`, {}, e);
+          logger.debug(
+            `Gossip validation ${type} rejected`,
+            {clientAgent, clientVersion, peerId: prettyPrintPeerIdStr(propagationSource)},
```

Comment:
slightly would prefer this order

```suggestion
            {peerId: prettyPrintPeerIdStr(propagationSource), clientAgent, clientVersion},
```